### PR TITLE
moved logging of LogicEngine from decisionengine logger to channel loggers

### PR DESCRIPTION
**Pull request by goodenou**
These changes address Issue #456. If we want the Sources, Transforms, etc to log to the channel loggers, a different set of changes needs to be made.

### CODE REVIEW

**Comment by jcpunk** [comment]
Can i have you run `pre-commit install` from the repository root? (and then `pre-commit run -a`) to see if that will auto correct the spacing issue?

File: src/decisionengine/framework/taskmanager/TaskManager.py [hunk 1]
```diff
@@ -65,7 +65,11 @@ def _create_module_instance(config_dict, base_class):
             class_name = _find_only_one_subclass(my_module, base_class)
 
     class_type = getattr(my_module, class_name)
-    return class_type(config_dict["parameters"])
+    #here we add the channel name to the LogicEngine class to get the correct logger
+    if class_name == "LogicEngine":
+        return class_type(config_dict["parameters"], channel_name)
+    else:
+        return class_type(config_dict["parameters"])
```
Comment (mambelli) [inline src/decisionengine/framework/taskmanager/TaskManager.py:72]: I'm wondering if the channel name should become part of the configuration or become something required for all the objects instantiated by the Taskmanager.
Please open a ticket for the next release to review (also in lieu of the sources changes) these changes (mention also the PR in the ticket), how working for a specific channel is made known to the worker (source/transform/logic engine).
There may be also the case of a worker re-used by multiple channels. I don't think this happens for the LogicEngine

**Review by mambelli — APPROVED** [review]
Approving and merging but please check the note and open a ticket. I know that an overall solution will require more work and considering the sources changes that will be in the next release

File: src/decisionengine/framework/taskmanager/TaskManager.py (hunk 1 above)
Comment (goodenou) [reply src/decisionengine/framework/taskmanager/TaskManager.py:72]: Hi Marco. Yes, I think the channel name should be something that is available through the configuration, i.e. in the config file. I will need to look at the worker code. Right now it is not clear to me how a task manager owned worker could be used by multiple channels.

**Pull request merged**